src/composables/useTrackers.ts

```diff
@@ -36,17 +36,15 @@ export function useTrackers(flawUuid: string, affects: ZodAffectType[]) {
     (affect) => affect.trackers.flatMap(tracker => tracker.affects)
   ));
 
-  const alreadyFiledTrackers = computed(() => affects.map(
-    (affect) => {
-      const maybeTracker = affect.trackers.find(
-        tracker => tracker.affects.some((affectUuid: string) => trackedAffectUuids.value.includes(affectUuid))
-      );
-      return maybeTracker && {
-        ...maybeTracker,
-        ...affect,
-      };
-    }
-  ).filter(Boolean));
+  const alreadyFiledTrackers = computed(() => affects.flatMap(
+    (affect) => affect.trackers.filter(
+      tracker => tracker.affects.some((affectUuid: string) => trackedAffectUuids.value.includes(affectUuid))
+    ).map(filedTracker => ({
+      ...filedTracker,
+      ...affect,
+    }))
+  ));
+    // .filter(Boolean));
```

Comment:
I find this a bit hard to read, I tried moving things around, wdyt? 
(same code but different formatting)
```js
const alreadyFiledTrackers = computed(() => {
  const filteredTrackers = affects.flatMap((affect) =>
    affect.trackers.filter((tracker) =>
      tracker.affects.some((affectUuid) =>
        trackedAffectUuids.value.includes(affectUuid)
      )
    )
  );

  return filteredTrackers.map((filedTracker) => ({
    ...filedTracker,
    ...affect,
  }));
});
```
Feel free to ignore the suggestion 😄 I think the problem is that cyclic reference of `affect.trackers & trackers.affects` that's making it a bit confusing, I don't know affects that well, and it took me a while to understand it

Reply:
+1 On this suggestion for readability